cli/internal/plugins/git/git.go

```diff
@@ -1,50 +1,11 @@
 package git
 
 import (
-	"bytes"
-	"context"
-	"os/exec"
-	"strings"
-
-	"github.com/gobuffalo/buffalo-cli/cli/internal/plugins/buildcmd"
 	"github.com/gobuffalo/buffalo-cli/plugins"
-	"github.com/gobuffalo/buffalo-cli/plugins/plugprint"
 )
 
-type Buffalo struct{}
-
-var _ buildcmd.Versioner = &Buffalo{}
-
-func (b *Buffalo) BuildVersion(ctx context.Context, root string) (string, error) {
-	if _, err := exec.LookPath("git"); err != nil {
-		return "", err
-	}
-
-	bb := &bytes.Buffer{}
-
-	cmd := exec.CommandContext(ctx, "git", "rev-parse", "--short", "HEAD")
-	cmd.Stdout = bb
-	if err := cmd.Run(); err != nil {
-		return "", err
-	}
-	s := strings.TrimSpace(bb.String())
-	if len(s) == 0 {
-		return "", nil
+func Plugins() []plugins.Plugin {
+	return []plugins.Plugin{
+		&Versioner{},
```

Comment:
Thanks, I need to master the difference between `Plugins` and `ScopedPlugins`. Will keep reading and adding code.